### PR TITLE
[Bugfix][Frame] Fix typos

### PR DESCRIPTION
**Pull request by ZenoTan**
## Checklist
Please feel free to remove inapplicable items for your PR.
- [x] The PR title starts with [$CATEGORY] (such as [NN], [Model], [Doc], [Feature]])
- [x] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented
- [x] To the my best knowledge, examples are either not affected by this change,
      or have been fixed to be compatible with this change
- [ ] Related issue is referred in this PR

## Changes

- Fix some typos in `python/dgl/frame.py`

### CODE REVIEW

File: python/dgl/frame.py [hunk 1]
```diff
@@ -183,7 +183,7 @@ def __setitem__(self, rowids, feats):
         feats : Tensor
             New features.
         """
-        self.update(idx, feats)
```
Comment (zheng-da) [inline python/dgl/frame.py:186]: why this kind of bug hasn't been triggered before?

Comment (ZenoTan) [reply python/dgl/frame.py:186]: Maybe previously code in `_deprecate` is used? I think we should add some tests on this.

Comment (yzh119) [reply python/dgl/frame.py:186]: I think they are used in the old `send_and_recv` functions which are no longer supported.

**Review by yzh119 — APPROVED** [review]
Thanks for your contribution.

File: python/dgl/frame.py (hunk 1 above)
Comment (ZenoTan) [reply python/dgl/frame.py:186]: Will it be triggered if someone updates graph feature manually?

**Pull request merged**